### DISCUSSION TITLE
Terrain作业运行程序时出现问题

### DISCUSSION THREAD

**Discussion by Grant-S-Z** [physics-data/faq#72] [Q&A] [answered]
运行程序时出现QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'，但是不影响图片正常显示。
但最后输出的pdf文件“无效或损坏”（已排除show的干扰）。

**Comment by heroxbd** [comment, marked as answer]
> 运行程序时出现QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'，但是不影响图片正常显示。

这与下面的问题无联系。

> 但最后输出的pdf文件“无效或损坏”（已排除show的干扰）。

输出 `png` 文件是否可以？

输出简单的图画出的 `pdf` 文件是否可以？

**Reply by Grant-S-Z** [reply]
后面的问题已解决，原因确实是需要渲染的点太密了，超出了pdf的能力范围。

**Reply by Harry-Chen** [reply]
> 后面的问题已解决，原因确实是需要渲染的点太密了，超出了pdf的能力范围。

这可能是你的查看器的问题，PDF 文件本身没有限制
